src/irr/video/COpenGLRenderpassIndependentPipeline.h

```diff
@@ -14,6 +14,63 @@ class COpenGLRenderpassIndependentPipeline : public IGPURenderpassIndependentPip
 public:
     using IGPURenderpassIndependentPipeline::IGPURenderpassIndependentPipeline;
 
+    struct SVAOHash
+    {
+        constexpr static size_t getHashLength()
+        {
+            return sizeof(hashVal)/sizeof(uint32_t);
+        }
+
+        bool operator!=(const SVAOHash& rhs) const 
```

Comment:
provide a `operator<` you are going to have so few elements in the cache that `unordered_map` will be slower than `map`

Reply:
decided to use vector, resolved